dev/src/user.clj

```diff
@@ -1,16 +1,17 @@
 (ns user
   (:require
    [hawk.assert-exprs]
+   [metabase.bootstrap]
```

Comment:
Unrelated, but it's important that this gets loaded on launch so the log config gets set up properly, otherwise if you start flitting around your REPL without calling `(dev)` it will use the wrong logging backend. Not 100% sure why but this seems to have started recently happening to me